### DISCUSSION TITLE
new controler game sir

### DISCUSSION THREAD

**Discussion by LeonTyrrell** [darthcloud/BlueRetro#1066] [Ideas]
hu here the trace for connecting gam sir controller
https://pt.aliexpress.com/item/1005006667923301.html?spm=a2g0o.order_list.order_list_main.40.40a8caa4cbTzvA&gatewayAdapt=glo2bra
[bt_trace game sir nova.zip](https://github.com/user-attachments/files/16772469/bt_trace.game.sir.nova.zip)

**Comment by darthcloud** [comment]
Can you try beta from #1031
